spring-cloud-zookeeper-discovery/src/main/java/org/springframework/cloud/zookeeper/discovery/configclient/ZookeeperConfigServerBootstrapper.java

```diff
@@ -53,10 +47,15 @@
 
 public class ZookeeperConfigServerBootstrapper implements BootstrapRegistryInitializer {
 
-	private static boolean isEnabled(Binder binder) {
-		return binder.bind(ConfigClientProperties.CONFIG_DISCOVERY_ENABLED, Boolean.class).orElse(false) &&
-				binder.bind(ConditionalOnZookeeperDiscoveryEnabled.PROPERTY, Boolean.class).orElse(true) &&
-				binder.bind("spring.cloud.discovery.enabled", Boolean.class).orElse(true);
+	private static PropertyResolver getPropertyResolver(BootstrapContext context) {
+		return context.getOrElse(PropertyResolver.class, new ConfigServerConfigDataLocationResolver.PropertyResolver(context.get(Binder.class), getBindHandler(context)));
+	}
+
+	private static boolean isEnabled(BootstrapContext bootstrapContext) {
```

Comment:
```suggestion
	public static boolean isEnabled(BootstrapContext bootstrapContext) {
```
Would it be possible to make the `isEnabled` method `public`, so that I could simply call it from my custom `BootstrapRegistryInitializer`? Currently the best option I have is to copy its source, but that could break (at runtime and in unexpected ways) if your implementation ever changes.